docs/en/ingest-management/elastic-agent/configuration/env/shared-env.asciidoc

```diff
@@ -462,6 +462,34 @@ contains your CA's certificate.
 
 // =============================================================================
 
+// tag::kibana-username[]
```

Comment:
Ah...the preview does not show up because you haven't included the tagged regions anywhere. You need to add include statements to any topics where you want the settings to appear. Look at container-envs.asciidoc for examples of how to include tagged regions.